protocol.html

```diff
@@ -906,10 +916,10 @@ <h3 property="schema:name">Security and Privacy Review</h3>
                     <dd about="#security-privacy-review-temporary-id" datatype="rdf:HTML" property="schema:description">The <cite><a href="#subscription-response" rel="cito:discusses">subscription response</a></cite> payload can contain a capability URL to protect the notification channel which is only exposed to authorized <cite><a href="#SubscriptionClient" rel="cito:discusses">Subscription Clients</a></cite>.</dd>
 
                     <dt about="#security-privacy-review-first-third-party" id="security-privacy-review-first-third-party"><a href="https://www.w3.org/TR/security-privacy-questionnaire/#first-third-party" rel="cito:repliesTo">How does this specification distinguish between behaviour in first-party and third-party contexts?</a></dt>
-                    <dd about="#security-privacy-review-first-third-party" datatype="rdf:HTML" property="schema:description">Inapplicable.</dd>
+                    <dd about="#security-privacy-review-first-third-party" datatype="rdf:HTML" property="schema:description">Not Applicable.</dd>
 
                     <dt about="#security-privacy-review-private-browsing" id="security-privacy-review-private-browsing"><a href="https://www.w3.org/TR/security-privacy-questionnaire/#private-browsing" rel="cito:repliesTo">How do the features in this specification work in the context of a browser’s Private Browsing or Incognito mode?</a></dt>
-                    <dd about="#security-privacy-review-private-browsing" datatype="rdf:HTML" property="schema:description">No different than <q>browser’s 'normal' state</q>.</dd>
+                    <dd about="#security-privacy-review-private-browsing" datatype="rdf:HTML" property="schema:description">No different than in the <q>browser’s 'normal' mode</q>.</dd>
```

Comment:
The reason I had proposed this change was because the question uses "mode". Also browser vendors tend to call it "mode" as well. Can you clarify where you have picked this quote from?

Reply:
The source is https://www.w3.org/TR/security-privacy-questionnaire/#private-browsing .

I've added the `cite` attribute linking to it.

I've also went ahead and proposed the change your suggesting here at the source: https://github.com/w3ctag/security-questionnaire/pull/150 . If that is accepted, we can update our spec. Until then, let's stick to the quote as is unless we want to express it differently - I don't think we should bother at this time.